### PR TITLE
add description to disk types

### PR DESCRIPTION
**Pull request by rafa-acioly**

### CODE REVIEW

**Comment by Akash4927** [comment]
@rafa-acioly Can you please provide the link of the issue in the description?

File: pkg/apis/openebs.io/v1alpha1/types.go [hunk 1]
```diff
@@ -27,13 +27,15 @@ type DiskSpec struct {
 	DevLinks []DiskDevLink `json:"devlinks,omitempty"` //DevLinks contains soft links of one disk
 }
 
+// DiskStatus represents a current state of the disk (Active/Inactive)
 type DiskStatus struct {
-	State string `json:"state"` //current state of the disk (Active/Inactive)
+	State string `json:"state"`
 }
 
+// DiskCapacity contains basic disk size in bytes and logical size in bytes
```
Comment (sagarkrsd) [inline pkg/apis/openebs.io/v1alpha1/types.go]: @rafa-acioly Can you please replace basic disk size -> disk capacity ?

File: pkg/apis/openebs.io/v1alpha1/types.go [hunk 2]
```diff
@@ -45,7 +47,7 @@ type DiskDetails struct {
 	FirmwareRevision string `json:"firmwareRevision"` // disk firmware revision
 }
 
-// DiskDevlink holds the maping between type and links like by-id type or by-path type link
+// DiskDevLink holds the maping between type and links like by-id type or by-path type link
```
Comment (sagarkrsd) [inline pkg/apis/openebs.io/v1alpha1/types.go]: Please replace maping -> mapping.

File: pkg/apis/openebs.io/v1alpha1/types.go [hunk 3]
```diff
@@ -27,13 +27,15 @@ type DiskSpec struct {
 	DevLinks []DiskDevLink `json:"devlinks,omitempty"` //DevLinks contains soft links of one disk
 }
 
+// DiskStatus represents a current state of the disk (Active/Inactive)
 type DiskStatus struct {
-	State string `json:"state"` //current state of the disk (Active/Inactive)
+	State string `json:"state"`
```
Comment (sagarkrsd) [inline pkg/apis/openebs.io/v1alpha1/types.go]: Field "State" should have comment also !

**Comment by kmova** [comment]
@rafa-acioly -- Thanks for attempting to fix this issue. Could you please respond to the review comments?

**Comment by akhilerm** [comment]
Closing PR due to inactivity.

**Pull request closed without merging**